trillian/migrillian/core/controller.go

```diff
@@ -248,6 +250,10 @@ func (c *Controller) verifyConsistency(ctx context.Context, root *types.LogRootV
 		}
 		return nil
 	}
+	if !c.opts.ConsistencyCheck {
```

Comment:
Can we add a test that this flag correctly prevents the code below from running.

Reply:
Unfortunately, there's no tests for this whole file. There is a bigger action item to add integration tests, including this bit.

Reply:
OK, that can be done later then.